content/Watsonx/WatsonxData/README.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Watsonx.data
+toc: false
+---
+
+# IBM Watsonx.Data Technical Sales Workshop
+
+This workshop covers the IBM Watsonx.Data L3 badge lab, highlighting 
```

Comment:
Looks like we need to finish the sentence here.  "highlighting...."   

Also, let's use the all lowercase in this spot and above on line 6.